src/main/io/osd.c

```diff
@@ -412,7 +415,8 @@ static void osdDrawSingleElement(uint8_t item)
         break;
 
     case OSD_HOME_DIST:
-        if (STATE(GPS_FIX) && STATE(GPS_FIX_HOME)) {
+//        if (STATE(GPS_FIX) && STATE(GPS_FIX_HOME)) {
+        if (STATE(GPS_FIX)) {
```

Comment:
Why is the `STATE(GPS_FIX_HOME)` condition removed?

Reply:
My intention was to get a home distance of a lost quad without the dangerous need to arm it again.
Practically didn't work. Could be reverted.

Reply:
`GPS_FIX_HOME` is set once when the first GPS fix is acquired and the craft is armed, if `STATE(GPS_FIX_HOME)` is `false` then there is no home location to calculate a distance from.

Having to rearm to get a valid distance is not required.